actions/promote_package.yaml

```diff
@@ -0,0 +1,40 @@
+---
+name: promote_package
+runner_type: mistral-v2
+description: Promote a package to production from staging.
+enabled: true
+entry_point: workflows/promote_package.yaml
+parameters:
+  package:
+    type: string
+    description: Package to promote
+    required: true
+  version:
+    type: string
+    description: Version to promote
+    required: true
+  revision:
+    type: string
+    description: Revision to promote
+    required: true
+  os:
+    description: OS to promote (will promote all if not specified)
+    required: true
+    default: all
```

Comment:
^^ It means if user promotes package with feature `A` with revision `N` with `all` or `deb` or `rpm` he can promote artifacts from different builds (eg. different features or bugs) for different OSes.

In reality, because of assumption, it can be really hard to find bug in production (packages from different builds).

^^ That looks like error-prone for me, if you plan to leave this process manual via ChatOps (not part of release 
machinery only).

From another hand, now I even think If we could use one revision number for all OSes, no matter what, to avoid any unsync.